systems/CustomCollision.lua

```diff
@@ -18,9 +17,7 @@ local function customCollision(entity)
         -- data is the player
         if data.isControlled then
             if CheckCollision(entity.body:getX(), entity.body:getY(), 32, 32, data.body:getX(), data.body:getY(), 32, 32) then
-                if entity.onCollision == "speedboost" then
-                    speedBoost.boost(data, entity);
-                end
+                entity.onCollision(data, entity)
```

Comment:
If the table is the first argument to one of its own methods you can use the shortcut syntax: `entity:onCollision(data)`.

Reply:
Done